src/main/resources/lib/enonic/react4xp/pageContributions/buildPageContributions.ts

```diff
@@ -42,7 +42,7 @@ export function buildPageContributions({
 	for (let i = 0; i < componentChunkUrls.length; i++) {
 		const componentChunkUrl = componentChunkUrls[i];
 		if (endsWith(componentChunkUrl, '.css')) {
-			pageContributions.headEnd.push(`<link href="${componentChunkUrl}" rel="stylesheet" type="text/css" />\n`);
+			pageContributions.headEnd.push(`<link href="${componentChunkUrl}" rel="stylesheet" type="text/css">\n`);
```

Comment:
I think we are in trouble here. Document type is defined DOCTYPE which we don't control.
If anyone wants XHTML, missing `/` will be an error. 
Extra `/` is a warning in html5 mode. We may have to have a configuration for that (and default to HTML5 as it the most common?)

Reply:
I reverted, and added a comment.
https://github.com/enonic/lib-react4xp/commit/3731e1c0f41aa246e301f40c78507d146f0bfca0